style.css

```diff
@@ -51,13 +60,30 @@ body {
 
 .menu__item {
   position: relative;
-  margin-right: 26px;
+  top: 47px;
+  margin-right: 24px;
   font: 500 19px/21px "Roboto", arial, helvetica, sans-serif;
   color: #000;
   text-decoration: none;
   transition: opacity .5s;
 }
 
+.menu__item:nth-child(1) {
+  margin-right: 19px;
+}
+
+.menu__item:nth-child(2) {
+  margin-right: 17px;
+}
+
+.menu__item:nth-child(3) {
+  margin-right: 21px;
+}
+
+ .menu__item:nth-child(4) {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)